apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -1118,3 +1120,44 @@ func getAnalyticsPVClaim(r *ReconcileAPI, deployVolumeMount []corev1.VolumeMount
 	}
 	return deployVolumeMount, deployVolume, err
 }
+
+func getTruststorePassword(r *ReconcileAPI) string {
+
+	var password string;
+	//get secret if available
+	secret := &corev1.Secret{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: truststoreSecretName, Namespace: wso2NameSpaceConst},
+	secret)
+	if err != nil && errors.IsNotFound(err){
+		encodedpassword := encodedTrustsorePassword
```

Comment:
Spelling mistake in `encodedTruststorePassword`

Reply:
Resolved